bot-data.json

```diff
@@ -165,6 +166,22 @@
                 "Uses hardcoded kick-offs made by Jeroen.",
                 "Only sees the ball."
             ]
+        },
+        "Bribblebot": {"name": "Bribblebot", "makers": ["L0laapk3"], "language": "python",
+            "statsUrl": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+            "notes": [
+                "Officially* RLBots most feared ballchaser.",
+                "Doesn't ever defend, as defending is a sign of being insecure. Instead, goes for demo's like a real alpha male.",
+                "Some of the code such as kickoffs and recovery code is not present in the botpack and public repo, as this code is stolen code from Bribblebots cousin Bottleneck."
+            ]
+        },
+        "Bottleneck": {"name": "Bottleneck", "makers": ["L0laapk3"], "language": "cpp",
+            "notes": [
+                "Has briefly shown up in one tournament, before retreating from further tournaments as this bot was severely unfinished.",
+                "Development originally started in python, but all the code has since been ported to cpp.",
+                "Is a completely multithreaded hivemind, complete with a separate thread for real time ball prediction.",
+                "Theres a rumor that this bot will never get finished."
```

Comment:
I like the notes like "Development originally started in python, but all the code has since been ported to cpp." and "Is a completely multithreaded hivemind, complete with a separate thread for real time ball prediction."

A lot of the other ones aren't very useful to a caster. You don't have to fix em, but I'll probably make a pass through everybody's notes at some point and trim it down to be brief and factual so casters can get info quickly.

Reply:
I tried to put stuff in there that could be interesting to say when the caster talk dies out